src/modules/promote.ts

```diff
@@ -26,11 +27,17 @@ const throwIfIsMaster = (workspace: string) => {
 }
 
 const handleConflict = async () => {
-  const conflictsFound = await checkForConflicts()
+  const conflictsFound = await vbase.checkForConflicts()
```

Comment:
I would rename it for something like `conflictsFoundForPagesGraphql` since the `checkForConflicts` is very specific for it. WDYT?

Reply:
Great, I can change it